			     BASH PATCH REPORT
			     =================

Bash-Release:	5.0
Patch-ID:	bash50-006

Bug-Reported-by:	Tomas Mozes <hydrapolic@gmail.com>
Bug-Reference-ID:	<CAG6MAzQumLU2vhnmr1UrYAUQAFW5Yo8hfM_sEibX9RJQWJkRrA@mail.gmail.com>
Bug-Reference-URL:	http://lists.gnu.org/archive/html/bug-bash/2019-03/msg00037.html

Bug-Description:

Bash-5.0 did not build successfully if SYSLOG_HISTORY was defined without
also defining SYSLOG_SHOPT.

--- a/builtins/shopt.def
+++ b/builtins/shopt.def
@@ -122,7 +122,7 @@ extern int assoc_expand_once;
 extern int array_expand_once;
 #endif
 
-#if defined (SYSLOG_HISTORY) && defined (SYSLOG_SHOPT)
+#if defined (SYSLOG_HISTORY)
 extern int syslog_history;
 #endif
 
--- a/patchlevel.h
+++ b/patchlevel.h
@@ -25,6 +25,6 @@
    regexp `^#define[ 	]*PATCHLEVEL', since that's what support/mkversion.sh
    looks for to find the patch level (for the sccs version string). */
 
-#define PATCHLEVEL 5
+#define PATCHLEVEL 6
 
 #endif /* _PATCHLEVEL_H_ */
